examples/stable/test/property/ic_api/call/test/dfx_generated/canister/canister.did

```diff
@@ -0,0 +1,3 @@
+service: () -> {
```

Comment:
All of the `dfx_generated` files should be completely removed, and once you add `dfx_generated` they will be gone. But please remove them from the git commit history as well.